### ISSUE TITLE
Crafting in Request Table does not fulfill HQM "Crafting" quests

### ISSUE THREAD

**Issue by codewarrior0** [RS485/LogisticsPipes#1074]
As above. 

`logisticspipes-0.9.3.128`
`HQM-The Journey-4.4.4a - CUSTOMIZED` (InfiTech 3.2.16 version)

**Comment by SonOfTheStars** [comment]
The Crafting Tables really arent made for actual Handcrafting. I dont know if they trigger the Same events as normal Crafters do.

**Comment by codewarrior0** [comment]
I don't expect the Crafting Tables to fulfill the quest. 

This issue is about the Request Table, which is not an automatic crafter. I took a peek at the Request Table's source. It will use a fake player for crafting (just like the Crafting Table) but it does this to make sure that any crafting byproducts that are left in the crafting grid or sent to the fake player's inventory are instead sent to the Request Table's inventory.

**Comment by SonOfTheStars** [comment]
OH, so its not registered as a standard crafting table
Thats probably fixable?
But the project is pretty frozen ATM...

**Comment by SonOfTheStars** [comment]
@codewarrior0 you dont happen to be a somewhat skilled modder :P

**Comment by codewarrior0** [comment]
Not really. My gut reaction was to fire a second onCrafting event for the player who is operating the Request Table, but then that's just going to cause all kinds of item dupes when other mods use the onCrafting event to dump byproducts into the player's inventory.

My second idea is to have the Request Table use the operating player for crafting, but then that sends stuff like empty buckets and crafting tools into the _player's_ inventory instead of the Table's, which is a real buzz kill.

My third idea is to add a soft depend on HQM and fire a crafting event directly into it.

**Comment by hron84** [comment]
> My second idea is to have the Request Table use the operating player for crafting, but then that sends stuff like empty buckets and crafting tools into the player's inventory instead of the Table's, which is a real buzz kill.

Actually most of the mods actually do this, or leave them in the grid (blocking the moving to the player's inventory). AE2, EnderIO's Inventory panel, etc I think because this is the only way to fire a global onCrafting event.

**Comment by Notlucky101** [comment]
All well said, but now the crafting in the request table is left in limbo now what?
![2019-07-26_07 44 26](https://user-images.githubusercontent.com/42447191/61960329-18e45980-af7a-11e9-8e56-bc6a74b8c06b.png)
if you look I order some ingots and the system does fine.
![2019-07-26_07 44 43](https://user-images.githubusercontent.com/42447191/61960381-2f8ab080-af7a-11e9-8d43-7524e1d52035.png)
but when I do fused quartz now it is a different matter
![2019-07-26_07 45 13](https://user-images.githubusercontent.com/42447191/61960427-4a5d2500-af7a-11e9-8228-24f606a8fc74.png)
it ends up in my inventory
![2019-07-26_07 45 42](https://user-images.githubusercontent.com/42447191/61960462-5ea12200-af7a-11e9-8c0b-6fea0fa5c5c5.png)
![2019-07-26_07 45 58](https://user-images.githubusercontent.com/42447191/61960477-64970300-af7a-11e9-9053-81c2e7c7d4ec.png)

Why? I would think if one thing works then why not the other order?
And having to restart the server every time this happens is a pain in the neck
